code/modules/mob/living/living_defense.dm

```diff
@@ -130,6 +130,8 @@
 /mob/living/proc/IgniteMob()
 	if(!CHECK_BITFIELD(datum_flags, DF_ISPROCESSING))
 		return FALSE
+	if(CHECK_BITFIELD(flags_pass, PASSFIRE))
+		return FALSE
```

Comment:
Passfire =/= fireresist

Reply:
Yep, we are still taking damage from the temperature, but we won't ignite. It's effetively different from fireresit and it's intended

Reply:
Ah, didn't think about that. Yeah, i'll change it